ansible/roles/common/defaults/main.yml

```diff
@@ -19,5 +19,6 @@ common_rpi_poe_fan_temp1: 50000 # Temperature in millicelcius
 common_rpi_poe_fan_temp2: 55000 # Temperature in millicelcius
 common_rpi_poe_fan_temp3: 60000 # Temperature in millicelcius
 
+log2ram_enabled: true # Set to false to skip installing log2ram
```

Comment:
Please prefix this with `common_` so its clear that this variable belongs to the `common` role. Ansible has no namespace concept so all variables are global by nature. This is our way to implement a poor man's namespacing.